sql-plugin/src/main/320+/scala/com/nvidia/spark/rapids/shims/Spark320PlusShims.scala

```diff
@@ -129,65 +129,6 @@ trait Spark320PlusShims extends SparkShims with RebaseShims with Logging {
       (cast, conf, p, r) => new CastExprMeta[Cast](cast,
         SparkSession.active.sessionState.conf.ansiEnabled, conf, p, r,
         doFloatToIntCheck = true, stringToAnsiDate = true)),
-    GpuOverrides.expr[AnsiCast](
-      "Convert a column of one type of data into another type",
-      new CastChecks {
```

Comment:
wait so didn't we essentially lose these for 3.4.0+.  I kind of expected these to move into the Cast function to run if ansiEnabled is true

Reply:
We still have it in `GpuCast ` with `ansiEnabled = true`, same as Spark 340+.
As you know, the backend of the GPU AnsiCast for Spark versions before 340 is `GpuCast(xxx, ansiEnabled = true)`.

Reply:
I must have been looking at the wrong version, I thought some of these checks were different between cast and ansi cast but rechecking it looks good.